fastlane/bin/🚀

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env ruby
+$LOAD_PATH.push File.expand_path("../../lib", __FILE__)
+
+require "fastlane"
+require "fastlane/commands_generator"
+Fastlane::CommandsGenerator.start
```

Comment:
We could, however I didn't have time to look into how to do that, in particular it would require some more testing with RubyGems and the bundled gem. We use the same technique as in fastlane in `gym` already.